download.py

```diff
@@ -0,0 +1,109 @@
+import sys
+import urllib.request
+import argparse
+import queue
+import os
+import urllib.request
+from concurrent.futures import ThreadPoolExecutor
+from pathlib import Path
+from PIL import Image
+from datetime import datetime
+
+
+start_time = datetime.now()
+
+
+def createParser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('urls_file', default='urllist.txt', nargs='?')
+    parser.add_argument('-d', '--dir', default='thumbnails/', nargs='?', type=str)
+    parser.add_argument('-t', '--threads', default=4, nargs='?', type=int)
+    parser.add_argument('-s', '--size', default='128x128', nargs='?')
+
+    return parser
+
+
+# Загрузка файлов
+def download_file(url, name):
+    """Переменные для статистики"""
+    global DOWNLOAD_COMPLETE, DOWNLOAD_ERROR
+    """Создаем папку"""
+    dirname = namespace.dir
+    if not os.path.isdir(namespace.dir):
+        os.mkdir(namespace.dir)
+
+    """Скачиваем файл"""
+    fname = name
+    try:
+        handle = urllib.request.urlopen(url)

```

Comment:
Please, use handle.length in order to count bytes downloaded

Reply:
Updated
Maybe so?
```python
    try:
        handle = urllib.request.urlopen(url)
        filepath = os.path.join(dirname, fname)

        length_in_bytes = os.stat(filepath).st_size
        SUM_BYTE += length_in_bytes
```